### PR TITLE
Set message explode time stories - desktop & mobile

### PR DESCRIPTION
**Pull request by buoyad**
Mobile coming soon. Also updates the icon font to get the bOoM! icon. r? @keybase/react-hackers 

'New' mode
image

'Not-new' mode
image

(note that previously a bunch of non-iconfont icons were left into the iconfont. I didn't skip that step while updating the font, hence the net -1,000LOC diff)

### CODE REVIEW

File: shared/chat/conversation/messages/set-explode-popup/index.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import {collapseStyles, globalColors, globalMargins, globalStyles} from '../../../../styles'
+
+const sortedDescriptions = messageExplodeDescriptions.sort((a, b) => (a.seconds < b.seconds ? 1 : 0))
+
+type HeaderProps = {
+  isNew: boolean,
+}
+
+const Header = (props: HeaderProps) => (
+  <Box style={headerContainerStyle}>
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/set-explode-popup/index.js]: lets start using Box2 and getting rid of these inner margins and use gap instead

File: shared/chat/conversation/messages/set-explode-popup/index.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+// @flow
+import * as React from 'react'
+import {messageExplodeDescriptions} from '../../../../constants/chat2'
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/set-explode-popup/index.js]: lets try and make these components very dumb. aka the descriptions should come in simply as an own prop (when integrated in the connector), and it'd be sorted, etc

**Comment by dabura667** [comment]
I like it!

Pumped!

**Comment by buoyad** [comment]
@keybase/react-hackers bump!

**Comment by buoyad** [comment]
@chrisnojima I don't think they were hooked into the code at all, they weren't added to `icon.constants`. I'm not sure what affect they had.

**Pull request merged**